addons/l10n_es_edi_tbai/tests/test_edi_xml.py

```diff
@@ -118,6 +118,20 @@ def test_xml_tree_post_multicurrency(self):
             xml_expected = self.with_applied_xpath(xml_expected_base, xpath)
             self.assertXmlTreeEqual(xml_doc, xml_expected)
 
+    def test_xml_tree_post_retention(self):
+        self.out_invoice.invoice_line_ids.tax_ids = [(4, self._get_tax_by_xml_id('s_irpf15').id)]
+        with freeze_time(self.frozen_today):
+            xml_doc = self.edi_format._get_l10n_es_tbai_invoice_xml(self.out_invoice, cancel=False)[self.out_invoice]['xml_file']
+            xml_doc.remove(xml_doc.find("Signature", namespaces=NS_MAP))
+            xml_expected_base = etree.fromstring(super().L10N_ES_TBAI_SAMPLE_XML_POST)
+            xpath = """
+                <xpath expr="//ImporteTotalFactura" position="after">
+                    <RetencionSoportada>600.00</RetencionSoportada>
+                </xpath>
+            """
+            xml_expected = self.with_applied_xpath(xml_expected_base, xpath)
```

Comment:
maybe it could be a good idea to also make sure that the invoice total doesn't include the retention tax amount?

Reply:
The invoice total is the node "ImporteTotalFactura" of the xml_expected_base (L10N_ES_TBAI_SAMPLE_XML_POST) an the total amount does not include the amount of the tax retention line added in the test, so testing the xml_doc with the xml_expected I think that allready ensures that retention taxes are ignored in both invoice line (ImporteTotal) and in the total ammount of the invoice (ImporteTotalFactura). Anny suggestions wpuld be appreciated.